packages/platform-browser/src/browser.ts

```diff
@@ -87,10 +87,21 @@ export const BROWSER_MODULE_PROVIDERS: StaticProvider[] = [
   {provide: SharedStylesHost, useExisting: DomSharedStylesHost},
   {provide: DomSharedStylesHost, useClass: DomSharedStylesHost, deps: [DOCUMENT]},
   {provide: Testability, useClass: Testability, deps: [NgZone]},
-  {provide: EventManager, useClass: EventManager, deps: [EVENT_MANAGER_PLUGINS, NgZone]},
+  {provide: EventManager, useFactory: useEventManager},
   ELEMENT_PROBE_PROVIDERS,
 ];
 
+export function useEventManager(): EventManager {
+  // This sorting is done because `DomEventsPlugin` supports all events (since its `supports()`
+  // method returns `true`). If some plugin appears in the array after `DomEventsPlugin`, then the
+  // check in the` _findPluginFor` method will not reach it.
+  // `-1` is used to make the plugin move to the beginning because the `EventManager` uses
+  // `reverse()` thus `DomEventsPlugin` will appear at the end.
+  const plugins = [...ɵɵinject(EVENT_MANAGER_PLUGINS)].sort(
+      plugin => plugin instanceof DomEventsPlugin ? -1 : 1);
+  return new EventManager(plugins, ɵɵinject(NgZone));
+}
```

Comment:
Just a quick comment: I believe you can use `useFactory` with the same `deps` (so you don't need to call internal `ɵɵinject` method):
```suggestion
  {provide: EventManager, useFactory: useEventManager, deps: [EVENT_MANAGER_PLUGINS, NgZone]},
  ELEMENT_PROBE_PROVIDERS,
];

export function useEventManager(plugins: EventManagerPlugin[], ngZone: NgZone): EventManager {
  // This sorting is done because `DomEventsPlugin` supports all events (since its `supports()`
  // method returns `true`). If some plugin appears in the array after `DomEventsPlugin`, then the
  // check in the` _findPluginFor` method will not reach it.
  // `-1` is used to make the plugin move to the beginning because the `EventManager` uses
  // `reverse()` thus `DomEventsPlugin` will appear at the end.
  const sortedPlugins = plugins.sort(plugin => plugin instanceof DomEventsPlugin ? -1 : 1);
  return new EventManager(sortedPlugins, ngZone);
}
```